src/Components/pages/Config.jsx

```diff
@@ -0,0 +1,130 @@
+  ]
+class Config extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+          Industry:null,
+          MarketCap:null,
+          Sector:null,
+          Rounds:null
+        };
+        this.submitHandler = this.submitHandler.bind(this);
+        this.getIndList = this.getIndList.bind(this);
+        this.getSectorList = this.getSectorList.bind(this);
+        this.getMarketCapList = this.getMarketCapList.bind(this);
+      }
+      // async componentDidMount() {  
+      //     this.setState({
+      //         Industries: await getIndustries(firestore),
+      //         Sectors: await getSectors(firestore),
+      //         MarketCaps: await getMarketCaps(firestore),
+      //     });
+      // }
+  
+      getIndList = async()=>{
+         
+          let IndList =await getIndustries(firestore);
+          return IndList;
+      }
+      getSectorList = async()=>{
+         
+          let SectorList =await getSectors(firestore);
+          return SectorList;
+      }
+      getMarketCapList = async()=>{
+         
+          let CapList =await getMarketCaps(firestore);
+          return CapList;
+      }
+      handleIndustryChange = selectedOption => {
+          this.setState({ Industry: selectedOption.value });
+          console.log(`Industry selected:`, selectedOption);
+      };
+  
+      handleSectorChange = selectedOption => {
+          this.setState({ Sector: selectedOption.value });
+          console.log(`Sector selected:`, selectedOption);
+      };
+  
+      handleMarketCapChange = selectedOption => {
+          this.setState({ MarketCap: selectedOption.value });
+          console.log(`MarketCap selected:`, selectedOption);
+      };
+      handleRoundChange = selectedOption =>{
+        this.setState({ Rounds: selectedOption.value });
+        console.log(`Rounds selected:`, selectedOption);
+      }
+      
+      submitHandler = async (event) => {
+          event.preventDefault(); // prevent page 
+          const data = new FormData(event.target);
+          console.log(data);
+          const Industry = this.state.Industry;
+          const Sector = this.state.Sector;
+          const MarketCaps = this.state.MarketCap;
+          const Rounds = this.state.Rounds;
+          let num = parseInt(data.get("NumSymbols"), 10);
+          const NumOfSymbols  = num ? num : 1;
+          let Symbols = []
+          try {
+            Symbols = await initSymbols(firestore,Industry,Sector,MarketCaps,NumOfSymbols);
+          }
+          catch(error) {
+            console.log("Error is "  + error)
+          }
+          //TODO: Separate initalize symbols and setuproom functions
+          if(Symbols.length)
+          {
+              let roomID =await setUpRoom(firestore,Symbols,Rounds,'');
+              this.props.updatePageType('created',roomID);
+          }
+          else
+          {
+              alert("No Symbols match your query");
+          }

```

Comment:
be consistent with braces formatting

```suggestion
          if(Symbols.length) {
              let roomID = await setUpRoom(firestore,Symbols,Rounds,'');
              this.props.updatePageType('created',roomID);
          }
          else {
              alert("No Symbols match your query");
          }
```